cmd/autoowners/main.go

```diff
@@ -390,7 +406,11 @@ func main() {
 		logrus.WithError(err).Fatal("Failed to change to root dir")
 	}
 
-	if err := pullOwners(gc, o.targetDir, sets.NewString(o.blacklist.Strings()...)); err != nil {
+	var configSubDirectories = o.configSubDirs.Strings()
+	if len(o.configSubDirs.Strings()) == 0 {
+		configSubDirectories = strings.Split(configSubDirs, ",")
+	}
+	if err := pullOwners(gc, filepath.Join(o.targetDir, o.targetSubDirectory), sets.NewString(o.blacklist.Strings()...), configSubDirectories, o.githubOrg, o.githubRepo); err != nil {
```

Comment:
nit: this is becoming hard to read with this many params. maybe extract the `Join` and `NewString` to named variables so that the call itself reads simpler?